lldb/source/Plugins/Language/Swift/SwiftHashedContainer.cpp

```diff
@@ -448,7 +448,6 @@ NativeHashedStorageHandler::NativeHashedStorageHandler(
       llvm::Optional<SwiftASTContextReader> scratch_ctx_reader = nativeStorage_sp->GetScratchSwiftASTContext();
       if (!scratch_ctx_reader)
         return;
-      SwiftASTContext *scratch_ctx = scratch_ctx_reader->get();
       auto *runtime = SwiftLanguageRuntime::Get(m_process);
```

Comment:
this one has the side-effect of acquiring the scratch context lock. But I would expect it to have been acquired by a parent call, so this should be safe.

Reply:
I'll keep this removal then.